Source/TCPError.swift

```diff
@@ -30,17 +30,17 @@ public enum TCPError: ErrorProtocol {
     case operationTimedOut(description: String, data: Data)
     case closedSocket(description: String)
 
-    static func lastReceiveErrorWithData(source: Data, bytesProcessed: Int) -> TCPError {
+    static func lastReceiveError(withData source: Data, bytesProcessed: Int) -> TCPError {
```

Comment:
I believe `Stream` stuff should just work and don't rely on Data. i vote for remove Data from error types :)